### ISSUE TITLE
Tetra Bluetooth connect/disconnect loop and "incorrect PIN"

### ISSUE THREAD

**Issue by ry755** [AsteroidOS/AsteroidOSSync#50]
Phone: Moto E4, Android 7.1.2 (LineageOS 14.1)
Watch: Sony Smartwatch 3, AsteroidOS 1.0

When I try to pair my watch with the sync app, the watch is spammed with notifications saying Connected, then saying Disconnected, then Connected, Disconnected, etc. This loop goes on forever unless I tap Unpair in the app.

If I try to pair it from the phone's Bluetooth settings it says, "Couldn't pair with tetra because of an incorrect PIN or passkey."

**Comment by DamageIncorporated** [comment]
In my impression this is very similar to #49, I also get multiple retries without ever completing pairing. 
Maybe the same problem?...

**Comment by ry755** [comment]
Yeah, it looks like it's the same problem, except mine doesn't show a pin on the watch's screen when trying to pair. It just immediately gets an error.

**Comment by FlorentRevest** [comment]
Hey, could you try [that](https://github.com/AsteroidOS/AsteroidOSSync/issues/49#issuecomment-390218862) as well @ry755 ? :)

**Comment by ry755** [comment]
Sure, I'll try that in a little while when I get home.

**Comment by moWerk** [comment]
Outdated, feel free to make a new issue if still relevant.
